src/store/printMyBarcode/actions.js

```diff
@@ -0,0 +1,75 @@
+import { handleResponse } from '@/api/ResponseHelper'
+
+const printJobV2 = async ({ getters }, params) => {
+  const request = getters.printJobV2Request
+
+  const labelTemplateName = getLabelTemplateName(getters)
+
+  const payload = createPrintJobJsonV2(
+    params.printer.text,
+    params.barcodesList,
+    params.copies,
+    labelTemplateName,
+  )
+
+  const promise = request.create({ data: payload })
+
+  const response = await handleResponse(promise)
+
+  if (!response.success) {
+    if (response.data.errors && response.data.errors.length != 0) {
+      let errors = response.data.errors.map((e) => e.source.pointer + ' ' + e.detail).join(', ')
+      return {
+        success: false,
+        errors: errors,
```

Comment:
I thought that at first as well, but I think as things stand we're only augmenting the errors with the pointer in cases where its avaialble, otherwise we fall back to the default error handling of the responseHandler.

There's a smell here definitely, but I think it might mean us needing to fix PMB errors.

Tempted to suggest we simplify this here at the expense of slightly funky errors, especially as we shouldn't be sending invalid payloads to PMB anyway.